tools/build/Library/ReleaseCreator.php

```diff
@@ -352,8 +352,8 @@ protected function setConfigDefinesConstants()
     {
         $configDefinesPath = $this->tempProjectPath . '/config/defines.inc.php';
         $configDefinesContent = file_get_contents($configDefinesPath);
-        $configDefinesNewContent = preg_replace('/(.*(define).*)_PS_MODE_DEV_(.*);/Ui', 'define(\'_PS_MODE_DEV_\', false);', $configDefinesContent);
-        $configDefinesNewContent = preg_replace('/(.*)_PS_DISPLAY_COMPATIBILITY_WARNING_(.*);/Ui', 'define(\'_PS_DISPLAY_COMPATIBILITY_WARNING_\', false);', $configDefinesNewContent);
+        $configDefinesNewContent = preg_replace('/(.*(define).*)["\']_PS_MODE_DEV_["\'](.*);/Ui', 'define(\'_PS_MODE_DEV_\', false);', $configDefinesContent);
```

Comment:
Previous regexp was too wide (not checking the quotes) so it matched on expression like `define('_PS_ENV_', _PS_MODE_DEV_ ? 'dev': 'prod');` just because `_PS_MODE_DEV_` was on the line